src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -699,9 +704,10 @@ protected DynamoModel(IStartConfiguration config)
                 // Do nothing for now
             }
 
+
             // If user skipped analytics from assembly config, do not try to launch the analytics client
             // or the feature flags client.
-            if (!areAnalyticsDisabledFromConfig && !Dynamo.Logging.Analytics.DisableAnalytics)
+            if (!areAnalyticsDisabledFromConfig && !Dynamo.Logging.Analytics.DisableAnalytics && !isServiceMode)
```

Comment:
@reddyashish @zeusongit This is purely a kick-start, let's brainstorm what can be skipped in service mode. Feel free to touch this branch with your ideas.

Reply:
good idea to validate these choices with profiling I guess.

Reply:
@saintentropy @mjkkirschner @BogdanZavu Any of you have profiled our WPFCLI? Any tips?